### PR TITLE
#162979790 Builds the Register Page for New Users

### PR DESCRIPTION
**Pull request by colMike**
## What does this PR do?
This pull request creates a register page on the Questioner Platform.

## Description of tasks to be completed
**The following are the tasks to be completed**

- Write html for register.html page
- Write CSS code for the registration page
- Ensure there is a link to the sign in page for those who already have an account

## How should you manually test this?
- Clone this repository on your local machine.
- cd UI/
- Open the register.html file on any browser. 

## Pivotal tracker stories associated with this PR:
#162979790

## Screenshots
register page

### CODE REVIEW

**Comment by colMike** [comment]
## What does this PR do?
This pull request creates a register page on the Questioner Platform.

## Description of tasks to be completed
**The following are the tasks to be completed**

- Write html for register.html page
- Write CSS code for the registration page
- Ensure there is a link to the sign in page for those who already have an account

## How should you manually test this?
- Clone this repository on your local machine.
- git checkout ft-user-registration-162979790
- cd UI/
- Open the register.html file on any browser. 

## Pivotal tracker stories associated with this PR:
#162979790

## Screenshots
register page

**Comment by Ogollah** [comment]
This is nice

File: UI/register.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        </div>
+        <p>Already have an account?<a href="login.html"><strong> Sign In</strong></a> </p>
+</div>
+
+
+<footer class="footer-dark">
+	<p id="copyright">Questioner. © 2019. All Rights Reserved</p>
+</footer>
+</body>
+</html>
```
Comment (Ogollah) [inline UI/register.html:58]: new line

**Comment by wachiranduati** [comment]
in your how should you manually test this....remove the `git checkout line`.....users need only cd into the folder and open the file in their browser ..... nice clean design nonetheless :)

**Comment by colMike** [comment]
Awesome...thanks, let me get rid of that.

**Pull request merged**